deploy/csi-daemonset.yaml

```diff
@@ -30,8 +30,8 @@ spec:
       # the same.
       containers:
       - name: mayastor-csi
-        image: mayadata/mayastor:develop
-        imagePullPolicy: Always
+        image: mayadata/mayastor:v1.0.0
+        imagePullPolicy: IfNotPresent
```

Comment:
Shouldn't we leave this as it is rather than overwriting it with the release version? Same for the other deploy YAML files.